cypress/utils/testing-data-utils.js

```diff
@@ -111,6 +111,81 @@ export const accessOrganization = (orgExternalId) => {
     }`,
   });
 };
+
+export const addDeviceToFacility = (facility, deviceIds) => {
+  return cy.makePOSTRequest({
+    operationName: "UpdateFacility",
```

Comment:
These utils are all copied over from our GraphQL queries and mutations by the same `operationName`